examples/react/index.js

```diff
@@ -39,3 +39,13 @@ const a = ( // eslint-disable-line no-unused-vars
 const Hello = () => <span>Hello</span> // eslint-disable-line no-unused-vars
 
 const Hello2 = () => <span>Hello 2</span> // eslint-disable-line no-unused-vars
+
+const OnAsHandlePrefix = (props) => (
```

Comment:
The name of the component is confusing. It's named like it's an eventHandler. I'll just change it to Hello3